<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>166</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curve-Based Rig Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>System Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="systemName_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="sysIn_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Curve:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="curve_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="curveIn_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mesh:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="mesh_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="meshIn_btn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>SkinCluster:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="skin_le"/>
      </item>
      <item>
       <widget class="QPushButton" name="skinIn_btn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;&lt;&lt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Controls Count:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="controlCount_sbx">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="useTips_chb">
      <property name="text">
       <string>Use Tips</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="createSofts_btn">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Place the amount of softMod in the curve&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Create SoftMods</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="softToJoint_btn">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Convert Soft-Mods to skinJoints( stickyPoints )&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>SoftMods To Joints</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="deleteSoft_btn">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete Soft-Mods&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Delete SoftMods</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="transWOnSel_btn">
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Transfer the weights of the selected softMod to the skin Joint&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Transfer Weight On Selected</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
